ramls/copycatimports.json

```diff
@@ -16,6 +16,10 @@
       "description": "profile to be used for importing",
       "type": "string"
     },
+    "selectedJobProfileId": {
```

Comment:
hello @RomanChernetskyi you can use "selectedJobProfileId": {
      "description": "Unique identifier",
      "$ref": "../../raml-util/schemas/uuid.schema"
    } for uuid types